config/flog.yml

```diff
@@ -1,2 +1,2 @@
 ---
-threshold: 27.7
+threshold: 33.8
```

Comment:
The number of inhabitants of a specific already existing context typically does not result in a big jump of the global scores.